src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -489,24 +490,6 @@
     You may use --role and --scopes to assign a specific role and narrow the scope to a resource or resource group.
     You may also use `az role assignment create` to create role assignments for this service principal later.
     See [steps to add a role assignment](https://aka.ms/azadsp-more) for more information.
-parameters:
-  - name: --cert
-    short-summary: Certificate to use for credentials.
-    long-summary: When used with `--keyvault,` indicates the name of the cert to use or create. Otherwise, supply a PEM or DER formatted public certificate string. Use `@{path}` to load from a file. Do not include private key info.
-  - name: --create-cert
-    short-summary: Create a self-signed certificate to use for the credential. Only the current OS user has read/write permission to this certificate.
-    long-summary: Use with `--keyvault` to create the certificate in Key Vault. Otherwise, a certificate will be created locally.
-  - name: --keyvault
-    short-summary: Name or ID of a KeyVault to use for creating or retrieving certificates.
-  - name: --years
-    short-summary: 'Number of years for which the credentials will be valid. Default: 1 year'
-  - name: --scopes
-    short-summary: >
-        Space-separated list of scopes the service principal's role assignment applies to.
-        e.g., /subscriptions/0b1f6471-1bf0-4dda-aec3-111122223333/resourceGroups/myGroup,
-        /subscriptions/0b1f6471-1bf0-4dda-aec3-111122223333/resourceGroups/myGroup/providers/Microsoft.Compute/virtualMachines/myVM
-  - name: --role
-    short-summary: Role of the service principal.
```

Comment:
Most of these arguments appear in all there commands: 

`ad app credential reset`
`ad sp credential reset`
`ad sp create-for-rbac`

So we define them in `_params.py` to avoid defining these thrice.